plugin/pkg/scheduler/scheduler.go

```diff
@@ -236,6 +233,9 @@ func (sched *Scheduler) scheduleOne() {
 			})
 			return
 		}
+		if err := sched.config.SchedulerCache.FinishBinding(&assumed); err != nil {
```

Comment:
Ok, thanks for explaining.  But if currState.pod.Spec.NodeName == pod.Spec.NodeName, then the pod will be deleted in ForgetPod() and I want to know on what condition the case  "currState.pod.Spec.NodeName != pod.Spec.NodeName" will be triggered.
Anyway, i moved the FinishBinding(...) back